web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -661,6 +661,12 @@ $.extend(fixmystreet.set_up, {
         } else {
             $submit.val($submit.data('valueOriginal'));
         }
+
+        // We might also have a response template to preselect for the new state
+        var $option = $("#report_inspect_form select.js-template-name option[data-problem-state='"+state+"']").first();
```

Comment:
Probably makes little/no difference, but could put `$('#report_inspect_form select.js-template-name')` in a variable first to use here and in if below.